cPP_DSC.adoc

```diff
@@ -2674,243 +2617,254 @@ FCS_CKM.5.1:: The TSF shall derive cryptographic keys with [*selection*: _key ty
+|[selection: 128, 192, 256] bits 
+|ISO/IEC 11770-6:2016 (subclause 7.3.4) [KPF4]
+
+NIST SP 800-108 Rev 1 (Section 4.1) [KDF in Double-Pipeline Iteration Mode] [selection: ISO/IEC 9797-1:2011 (CMAC), NIST SP 800-38B (CMAC)] [selection: ISO/IEC 18033-3:2010 (AES), FIPS PUB 197 (AES), ISO/IEC 18033-3:2010 (subclause 4.5) [HIGHT], ISO/IEC 29192-2:2019 (subclause 6.3) [LEA], ISO/IEC 18033-3:2010 (subclause 5.4) [SEED]],
+
+[selection: ISO/IEC 9797-2:2021 (HMAC), FIPS PUB 198-1 (HMAC)],
 
-{empty}[*selection*: [.underline]#ISO/IEC 9797-1:2011 (CMAC), NIST SP 800-38B (CMAC), ISO/IEC 18033-3:2010 (AES), ISO/IEC 9797-2:2021 (HMAC), FIPS PUB 198-1 (HMAC), ISO/IEC 10118-3:2018 (SHA), FIPS PUB 180-4 (SHA)]#
+[selection: ISO/IEC 10118-3:2018 (SHA), FIPS PUB 180-4 (SHA)]
 
 |KDF-XOR
 |More than one intermediary keys
 |exclusive OR (XOR)
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits 
+|[selection: 128, 192, 256] bits 
 |N/A
 
 |KDF-ENC
 |Two keys 
 |Encrypting using an algorithm specified in FCS_COP.1/SKC
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits 
+|[selection: 128, 192, 256] bits 
 |N/A
 
 |KDF-HASH 
 |Shared secret
 |Hash function from FCS_COP.1/Hash 
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits 
+|[selection: 128, 192, 256] bits 
 |NIST SP 800-56C Rev 2 (Section 4, Option 1)
 
 |KDF-MAC-1S
 |Shared secret, salt, output length, fixed information 
 |Keyed Hash function from FCS_COP.1/KeyedHash
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits 
+|[selection: 128, 192, 256] bits 
 |NIST SP 800-56C Rev 2 (Section 4, Options 2, 3)
 
 |KDF-MAC-2S 
 |Shared secret, salt, IV, output length, fixed information 
 |[MAC Step]
 
-{empty}[*selection*: [.underline]#AES-128-CMAC; AES-192 -CMAC; AES-256 -CMAC; HMAC-SHA-1; HMAC-SHA-256; HMAC-SHA-512]# as the EEF and;
+[selection: AES-128-CMAC, AES-192-CMAC, AES-256-CMAC, HMAC-SHA-1, HMAC-SHA-256, HMAC-SHA-512] as the EEF and;
 
 [KDF Step] 
 
-{empty}[*selection*: [.underline]#KDF-CTR, KDF-FB, KDF-DPI]# using {empty}[*selection*: [.underline]#AES-128-CMAC; AES-192-CMAC; AES-256-CMAC; HMAC-SHA-1; HMAC-SHA-256; HMAC-SHA-512]# as PRF.
+[selection: KDF-CTR, KDF-FB, KDF-DPI] using [selection: AES-128-CMAC, AES-192-CMAC, AES-256-CMAC, HMAC-SHA-1, HMAC-SHA-256, HMAC-SHA-512] as PRF
 
-|{empty}[*selection*: [.underline]#128, 192, 256]# bits 
+|[selection: 128, 192, 256] bits 
 |NIST SP 800-56C Rev 2 (Section 5)
 
-{empty}[*selection*: [.underline]#ISO/IEC 9797-1:2011 (CMAC), NIST SP 800-38B (CMAC), ISO/IEC 18033-3:2010 (AES), ISO/IEC 9797-2:2021 (HMAC), FIPS PUB 198-1 (HMAC), ISO/IEC 10118-3:2018 (SHA), FIPS PUB 180-4 (SHA)]#
+[selection: ISO/IEC 9797-1:2011 (CMAC), NIST SP 800-38B (CMAC), ISO/IEC 18033-3:2010 (AES), FIPS PUB 197 (AES), ISO/IEC 9797-2:2021 (HMAC), FIPS PUB 198-1 (HMAC), ISO/IEC 10118-3:2018 (SHA), FIPS PUB 180-4 (SHA)] 
+
+|KDF-KMAC
+|See NIST 800-108
+|[selection: KMAC128, KMAC256, KMACXOF128, KMACXOF256]
```

Comment:
SP 800-108 Rev 1 explicitly does not allow KMACXOF128 and KMACXOF256 for this KDF:
> For KMACXOF128 and KMACXOF256, L is not an input for the function but a parameter to determine the output length. However, for a key- derivation function to comply with this Recommendation, the length of the derived keying material must be known before the function is called. Therefore, KMAC128 or KMAC256 shall be used for the key-derivation function.

Reply:
@slpotte

Reply:
Agreed.  Removing from catalog.

Reply:
removed